modules/priceFloors.ts

```diff
@@ -103,7 +121,33 @@ const getHostname = (() => {
   }
 })();
 
-// First look into bidRequest!
+/**
+ * @summary Check if a bidRequest contains any user IDs from the specified tiers
+ * Returns an object with keys like 'userId.tierName' with boolean values (0/1)
+ */
+export function resolveTierUserIds(tiers, bidRequest) {
+  if (!tiers || !bidRequest?.userIdAsEid?.length) {
+    return {};
+  }
+
+  // Get all available EID sources from the bidRequest (single pass)
+  const availableSources = bidRequest.userIdAsEid.reduce((acc: Set<string>, eid: { source?: string }) => {
```

Comment:
I think both of these are checking for user ids wrong and would cause this whole setup to never work?
instead of `bidRequest.userIdAsEid` it should be the plural `bidRequest.userIdAsEids`